app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -62,7 +62,12 @@ public void loadAttendees(boolean forceReload) {
         attendeesView.showScanButton(false);
 
         eventRepository.getAttendees(eventId, forceReload)
-            .toList()
+            .toSortedList((a, b) -> {
+                int f, l;
+                return (f = a.getFirstName().toLowerCase().compareTo(b.getFirstName().toLowerCase())) == 0 ? (l = a.getLastName().toLowerCase().compareTo(b.getLastName().toLowerCase())) == 0 ? a.getEmail().toLowerCase().compareTo(b.getEmail().toLowerCase()) : l : f;
+            })
+            .subscribeOn(Schedulers.computation())
+            .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Hmm, ok. Just makes it a little verbose. You can at least remove the observeOn part